quarterback-packages/api/src/routes/snapshot/snap.ctrl.ts

```diff
@@ -70,27 +69,14 @@ export const saveSnapshot = async (
   next: NextFunction
 ) => {
   try {
-    const result = await snapService.saveSnapshot(req.body)
-    if ('data' in result) {
-      res.json({ snapshot: result.data })
-    } else {
-      next(new CustomError(result.err, result.code))
+    const payload: ISaveSnapshotRequest = req.body
+    const document: any = await docService.findDocument(payload.docId)
+    if (!document) {
+      next(new CustomError("Document not found"))
     }
-  } catch (err) {
-    next(err)
-  }
-}
-
-export const updateSnapshot = async (
-  req: AuthRequest<IUpdateSnapshotRequest, { snapshotId: string }>,
-  res: AuthResponse,
-  next: NextFunction
-) => {
-  try {
-    const { snapshotId } = req.params
-    const result = await snapService.updateSnapshot(snapshotId, req.body)
+    const result = await snapService.saveSnapshot(payload, document.data.doc)
```

Comment:
@mbartenev-atypon this is something I discussed with Hashem, and I want to make sure you're on the same page or let us know if there is an objection.

When creating snapshots, quarterback will no longer receive the document in the request, and instead will lookup the document by id.